scripts/py_matter_idl/matter_idl/test_matter_idl_parser.py

```diff
@@ -774,6 +774,22 @@ def test_emits_events(self):
 
         self.assertEqual(actual, expected)
 
+    def test_revision(self):
+        actual = parseText("""
+            server cluster A = 1 { } // revision 1 implied
+            client cluster B = 2 { revision 1; }
+            client cluster C = 3 { revision 2; }
+            client cluster D = 4 { revision 123; }
+        """)
+
+        expected = Idl(clusters=[
+            Cluster(side=ClusterSide.SERVER, name="A", code=1, revision=1),
+            Cluster(side=ClusterSide.CLIENT, name="B", code=2, revision=1),
+            Cluster(side=ClusterSide.CLIENT, name="C", code=3, revision=2),
+            Cluster(side=ClusterSide.CLIENT, name="D", code=4, revision=123),
```

Comment:
I guess you are racing with yourself https://github.com/project-chip/connectedhomeip/pull/30605